packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -934,7 +937,7 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
     }
 
     // clear all of the pending info
-    this._clearPendingInfo();
+    !this.props.multiSelect && this._clearPendingInfo();
```

Comment:
>multiSelect [](start = 16, length = 11)

The change on this line looks a little scary... we'll need to make sure to validate this well

Reply:
(and below)

---
In reply to: [355044289](https://github.com/OfficeDev/office-ui-fabric-react/pull/11397#discussion_r355044289) [](ancestors = 355044289)

Reply:
maybe we need an additional state to track the currently focused control